tests/test_wps_climdex_ptot.py

```diff
@@ -27,7 +26,23 @@ def buil_params(climdex_input, ci_name, threshold, vector_name, output_file):
         (local_path("climdexInput.rda"), "ci", 99, "vector_name"),
     ],
 )
-def test_wps_climdex_ptot(climdex_input, ci_name, threshold, vector_name):
+def test_wps_climdex_ptot_rda(climdex_input, ci_name, threshold, vector_name):
+    with NamedTemporaryFile(
+        suffix=".rda", prefix="output_", dir="/tmp", delete=True
+    ) as out_file:
+        datainputs = buil_params(
+            climdex_input, ci_name, threshold, vector_name, out_file.name
+        )
+        run_wps_process(ClimdexPtot(), datainputs)
+
+
+@pytest.mark.parametrize(
+    ("climdex_input", "ci_name", "threshold", "vector_name"),
+    [
+        (local_path("climdexInput.rds"), "ci", None, "vector_name"),
+    ],
+)
+def test_wps_climdex_ptot_rds(climdex_input, ci_name, threshold, vector_name):
     with NamedTemporaryFile(
         suffix=".rda", prefix="output_", dir="/tmp", delete=True
     ) as out_file:
```

Comment:
I'm not sure these tests need to be separate, we could very easily put these cases together.

Reply:
Ya again this is left over from the implementation with 2 inputs and I missed it, thanks for pointing it out